packages/core/src/rules/arazzo/index.ts

```diff
@@ -0,0 +1,11 @@
+import { ArazzoRule } from '../../visitors';
+import { Spec } from '../common/spec';
+import type { ArazzoRuleSet } from '../../oas-types';
+import { Assertions } from '../common/assertions';
+
+export const rules: ArazzoRuleSet<'built-in'> = {
+  spec: Spec as ArazzoRule,
+  assertions: Assertions,
+};
+
+export const preprocessors = {};
```

Comment:
Do you use it?

Reply:
No where - just template for the future

Reply:
Then I don't think it makes sense to create that now (especially to create preprocessors in the rules folder -- I doubt anyone will be looking there).

Reply:
I keep it in sync with other folders - it just mean that there are no preprocessors | decorators for now - but at least we have a placeholder ready for this.
If you strictly against, I can remove it, have those settings to keep in sync with other types like oas3, async .....